src/backend/tasks_io/datafeeds/datafeed_fms_api.py

```diff
@@ -415,12 +389,12 @@ def _get_event_short(self, event_short: str, event: Optional[Event]) -> str:
 
         # Second, check if it is a new 2023 division event code
         if event and event.year == 2023:
-            return DatafeedFMSAPI.EVENT_SHORT_EXCEPTIONS_2023.get(
+            return Event.EVENT_SHORT_EXCEPTIONS_2023.get(
```

Comment:
Is this additional logic necessary anymore? Can we just use `event.first_api_code`?